tests/arg_parse/test_arg_parse_version_check.h

```diff
@@ -63,7 +63,7 @@ const std::string TestVersionCheck_::APP_VERSION_FILENAME     = TestVersionCheck
                                                                 static_cast<std::string>(".version");
 const std::string TestVersionCheck_::APP_TIMESTAMP_FILENAME   = TestVersionCheck_::PATH + "/" +
                                                                 TestVersionCheck_::APP_NAME +
-                                                                static_cast<std::string>("_release.timestamp");
+                                                                static_cast<std::string>("_usr.timestamp");
```

Comment:
No since it's only a name difference and no change in functionality?
I could choose either one and I chose to use the usr.timestamp filename.